llvm/test/Transforms/PGOProfile/instr-gen-cold-function.ll

```diff
@@ -0,0 +1,24 @@
+; RUN: opt < %s  --passes=pgo-instr-gen -instrument-cold-function-coverage -S  | FileCheck --check-prefixes=COLD %s
+; RUN: opt < %s  --passes=pgo-instr-gen -instrument-cold-function-coverage -cold-function-coverage-max-entry-count=1 -S  | FileCheck --check-prefixes=ENTRY-COUNT %s
+
+; COLD: call void @llvm.instrprof.increment(ptr @__profn_foo, i64  [[#]], i32 1, i32 0)
```

Comment:
Do you plan to support `-instrument-cold-function-coverage` without `-pgo-function-entry-coverage`? If not, I think we should add the flag here so we get the `llvm.instrprof.cover` intrinsic to more closely align with expected behavior.